Resources/config/ckeditor.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="cmf_content.form.extension.ckeditor"
+            class="Symfony\Cmf\Bundle\ContentBundle\Form\Extension\CKEditorExtension">
```

Comment:
should we add something to configure editor options in the content bundle? i would say that would be useful, kind of as we had before, only not looped through sonata admin.